src/app/shipping/MultiShippingForm.tsx

```diff
@@ -87,32 +105,108 @@ class MultiShippingForm extends PureComponent<MultiShippingFormProps & WithLangu
         }
 
         return (
-            <Form>
-                <ul className="consignmentList">
-                    { items.map(item => (
-                        <li key={ item.key }>
-                            <ItemAddressSelect
-                                addresses={ addresses }
-                                item={ item }
-                                onSelectAddress={ this.handleSelectAddress }
-                                onUseNewAddress={ onUseNewAddress }
-                            />
-                        </li>
-                    )) }
-                </ul>
-
-                <ShippingFormFooter
-                    cartHasChanged={ cartHasChanged }
-                    isLoading={ isLoading }
-                    isMultiShippingMode={ true }
-                    shouldDisableSubmit={ this.shouldDisableSubmit() }
-                    shouldShowOrderComments={ shouldShowOrderComments }
-                    shouldShowShippingOptions={ !hasUnassignedLineItems(consignments, cart.lineItems) }
+            <Fragment>
+                <ErrorModal
+                    error={ createCustomerAddressError }
+                    message={
```

Comment:
There's only one error at the moment so what you have is good. But if in the future we have more than one error, we could do something like https://github.com/bigcommerce/checkout-js/blob/0d34c09f6b324921e2a9c4958bbdaab6382043de/src/app/payment/Payment.tsx#L206 where we can map to a different error message depending on its type.